benchmark/rclnodejs/client-endurance-test.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2018 Intel Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const rclnodejs = require('../../index.js');
+
+rclnodejs.init().then(() => {
+  console.log('The client will send a SetBool request every 100ms in 24 hours.');
+  console.log('Beginning');
+
+  const node = rclnodejs.createNode('endurance_client_rclnodejs');
+  const client = node.createClient('std_srvs/srv/SetBool', 'set_flag');
+  let count = 0;
+
+  let timer = setInterval(() => {
+    if (count > 864000) {
+      clearInterval(timer);
+      rclnodejs.shutdown();
+      console.log('Done');
+    } else {
+      client.sendRequest(true, (response) => {
+        count++;
```

Comment:
When the `response` could not been received in 100ms, `setInterval` will take priority to go into next loop, which may result in the overall execution time exceeds 24 hours.